test/CMakeLists.txt

```diff
@@ -269,6 +269,10 @@ if(UMF_POOL_SCALABLE_ENABLED AND (NOT UMF_DISABLE_HWLOC))
         NAME scalable_pool
         SRCS pools/scalable_pool.cpp malloc_compliance_tests.cpp
         LIBS ${UMF_UTILS_FOR_TEST} ${UMF_BA_FOR_TEST})
+    add_umf_test(
+        NAME test_pool_null_params
```

Comment:
if we have ifdefs (proposed in the comment below) we could move this test out of the `if` here